scripts/cloud-environment-setup.sh

```
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+# LOCAL AGENTS: If you are not a cloud agent, do not execute this script.
+# Manual environment setup for Codex and Claude Code cloud tasks.
+# Configure the provider's setup command as: bash scripts/cloud-environment-setup.sh
+
+set -euo pipefail
+
+readonly UV_VERSION="0.12.5"
+readonly JUST_VERSION="1.58.0"
+readonly BACKLOG_VERSION="1.50.1"
+readonly LOCAL_BIN="${HOME}/.local/bin"
+readonly SCRIPT_DIR="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" && pwd)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)